packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -208,36 +208,53 @@ export class Dropdown extends BaseComponent<IDropdownProps, IDropdownState> {
         this.setState({
           isOpen: !this.state.isOpen
         });
+
+        ev.stopPropagation();
```

Comment:
Would be cleaner with having a flag `eventHandled` and set it appropriately in switch cases. In the end we can stop event propagation if event was not handled.